schoolopy/main.py

```diff
@@ -1891,15 +1893,44 @@ def get_assignment_comment(self, section_id, assignment_id, comment_id):
     # TODO: Support Web Content Package
     # TODO: Support Completion
 
+    def get_section_grades(self, section_id, query_name=None, query_value=None):
+        """
+        Returns a list of grades (paged).
+
+        :param section_id: ID of course's section.
+        :return: a list of grades for that section.
+
+        The following optional query strings can be appended to the path
+        to filter results:
+
+        :param query_name (timestamp, assignment_id, or enrollment_id)
```

Comment:
This is a pretty non-standard docstring style. Make sure that your notation matches the others.